.github/workflows/build.yml

```diff
@@ -31,7 +31,7 @@ jobs:
       - uses: actions/checkout@master
 
       - name: Setup Node.js
-        uses: actions/setup-node@v2
+        uses: actions/setup-node@v3
         with:
           node-version: ${{ matrix.node_version }}
```

Comment:
the breaking change removed the `version` attribute in favor of `node-version`, which we already use.